src/libp2p/connectionsManager.ts

```diff
@@ -37,6 +37,7 @@ interface IConstructor {
   agentHost: string
   options?: Partial<ConnectionsManagerOptions>
   io: any
+  isWaggleMobileMode?: boolean
```

Comment:
Couldn't it be in options?

Reply:
It would be consistent to passing other extra flags and options are already being passed to Storage